src/components/Settings/preferences/accordions/MonthlyGoalAccordion/MonthlyGoalAccordion.tsx

```diff
@@ -105,10 +107,15 @@ export const MonthlyGoalAccordion: React.FC<MonthlyGoalAccordionProps> = ({
   };
 
   const instructions = calculatedGoal
-    ? t(
-        'Based on the past year, NetSuite estimates that you need at least {{goal}} of monthly support. You can use this amount or choose your own target monthly goal.',
-        { goal: formattedCalculatedGoal },
-      )
+    ? initialMonthlyGoal === null
+      ? t(
+          'Based on the past year, NetSuite estimates that you need at least {{goal}} of monthly support. You can choose your own target monthly goal or leave it blank to use the estimate.',
+          { goal: formattedCalculatedGoal },
+        )
+      : t(
+          'Based on the past year, NetSuite estimates that you need at least {{goal}} of monthly support. You can choose your own target monthly goal or reset it to the estimate.',
+          { goal: formattedCalculatedGoal },
+        )
```

Comment:
This isn't required, but I wonder if there is a way to make this double if-else easier to read.